### ISSUE TITLE
Filter ATAC before running scDblFinder

### ISSUE THREAD

**Issue by ccruizm** [plger/scDblFinder#76]
Good day,

I wanna use `scDblFinder` on my scATAC data. For RNA, you warn users to perform initial QC so it does not influence the modeling of doublets for more precise doublet calling.

I have checked the ATAC vignette but could not find information about this particular point. What would you recommend?

I appreciate any suggestions you can give me.

**Comment by plger** [comment]
Hi,

Actually it turns out that scDblFinder and its thresholding procedure are pretty robust to prior filtering or not, so I don't think the results will change much (unless you're filtering out a majority of cells).
The same therefore applies for ATAC-seq when using the [aggregation-based method](https://plger.github.io/scDblFinder/articles/scATAC.html#applying-the-scdblfinder-method).

Regarding the [Amulet method](https://plger.github.io/scDblFinder/articles/scATAC.html#using-the-amulet-method), it's anyway is largely independent across cells and hence of prior filtering (the only impact across cells is the filtering out of possible repeat regions).

So in conclusion: it shouldn't make much of a difference.

**Comment by ccruizm** [comment]
Thanks for the speedy reply and for clarifying this! It is good to know that it won't impact the performance of the doublet calling.
